src/StatusQ/Controls/StatusComboBox.qml

```diff
@@ -0,0 +1,145 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+import QtGraphicalEffects 1.13
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Popups 0.1
+
+import QtQuick.Templates 2.14 as T
+
+Item {
+    id: root
+
+    property alias comboBox: comboBox
+    property alias model: comboBox.model
+    property alias delegate: comboBox.delegate
+    property alias label: labelItem.text
+    property alias validationError: validationErrorItem.text
+
+    implicitWidth: layout.implicitWidth
+    implicitHeight: layout.implicitHeight
+
+    ColumnLayout {
+        id: layout
+
+        anchors.fill: parent
+        spacing: 0
+
+        StatusBaseText {
```

Comment:
Hmm, just wondering what is this text above the combo used for? Is this a frequent usage pattern? Otherwise it would make much more sense to have `T.ComboBox` as the root item :(

Reply:
Ok